resources/views/api/emails/voucher_duplicate_email_textonly.blade.php

```diff
@@ -0,0 +1,22 @@
+* Voucher Duplicate Entered *
+
+==============================================
+
+[xXx]
+Hi {{ config('mail.to_admin.name') }},
+
+{{ $user }} has tried to submit voucher
+{{ $vouchercode }} against
+{{ $trader }} of
+{{ $market->name ?? 'no associated market' }}'s account, however that voucher has already been submitted by another trader.
```

Comment:
Please add to app/Mail/VoucherDuplicateEnteredEmail.php instead so you don't need to duplicate.